### DISCUSSION TITLE
Leeway PGN 128000

### DISCUSSION THREAD

**Discussion by HBrew62** [canboat/canboat#228] [General]
Hi Kees
I noticed that your definition of this PGN is unsigned. However, on this site [https://www.nmea.org/Assets/20170204%20nmea%202000%20leeway%20pgn%20final.pdf](url)
that Leeway is a signed value with positive values leeway to starboard and negative to port

**Comment by keesverruijt** [comment]
This was addressed in last year's big rewrite.
